addons/stock/wizard/stock_track_confirmation_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_stock_track_confirmation" model="ir.ui.view">
+        <field name="name">stock.track.confirmaton.view.form</field>
+        <field name="model">stock.track.confirmaton</field>
+        <field name="arch" type="xml">
+            <form string="Confirmation Warning about lot/serial number is empty">
```

Comment:
maybe "Lots or serial numbers were not provided to tracked products"

Reply:
change it